src/scss/components/_inputs.scss

```diff
@@ -84,7 +84,9 @@ $input-select-margin-right: 1;
 
 .usa-checkbox__input:checked + .usa-checkbox__label::before,
 .usa-checkbox__input:checked:disabled + .usa-checkbox__label::before {
+  background-color: color('disabled');
   background-size: units(1) auto;
+  box-shadow: inset 0 0 0 units($border-width) color('disabled');
```

Comment:
The background color and box shadow styling here should be limited to disabled fields. Currently, it's applying to all:

![image](https://user-images.githubusercontent.com/1779930/168612266-58708a09-e5b8-4b12-8cff-3451a4bbcd60.png)